types/overload-protection/index.d.ts

```diff
@@ -0,0 +1,51 @@
+// Type definitions for overload-protection 1.2
+// Project: https://github.com/davidmarkclements/overload-protection
+// Definitions by: Daniel Hirth <https://github.com/dornfeder>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare namespace protect {
+    type KoaFrameworkSelection = 'koa';
+    type HttpFrameworkSelection = 'express' | 'http' | 'restify';
+
+    interface ProtectionConfig {
+        production?: boolean;
+        clientRetrySecs?: number;
+        sampleInterval?: number;
+        maxEventLoopDelay?: number;
+        maxHeapUsedBytes?: number;
+        maxRssBytes?: number;
+        errorPropagationMode?: boolean;
+        logging?: boolean | string | (() => void);
```

Comment:
I'm assuming the function is supposed to be a logging function.  If that's the case, should it accept arguments?

Reply:
Ah, i missed that! I added a msg parameter to the function. Thanks.